yb_migrate/cmd/importData.go

```diff
@@ -1035,6 +1023,29 @@ func getCopyCommand(table string) string {
 	return "" // no-op
 }
 
+func setProgressAmount(filePath string, progressAmount int64) {
+	log.Infof("set user.progress_amount=%d for file %q", progressAmount, filePath)
```

Comment:
This message will flood the logs. Please reduce its log level to `Debugf`.

Reply:
Sure, nice catch.